client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -15,171 +9,93 @@ import React from 'react';
-      tags: ['burgers', 'startup', 'non-GMO', 'food distributor'],
-      mission: 'To promote and make known the universal appeal of burgers.',
-      description:
-        "Isabel's burgers is a startup food distributor operating locally in Auckland, New Zealand. We service restaurants, retail stores, and educational institutions. We focus on burgers, but also deliver and distribute a range of other non-GMO foods. We are able to offer vegan options for all our burgers, including those made with synthetic meats.",
-      profilePicture: burger,
-      instagram: 'https://instagram.com',
-      facebook: 'https://facebook.com',
-    },
-    {
-      name: "Zaid's Chickens",
-      location: {
-        address: 'Tracy, CA',
-        lat: 37.6915, // latitude (calculated with Geocoding)
-        lng: -121.4348, // longitude (calculated with Geocoding)
-      },
-      email: 'yummychicken@gmail.com',
-      phone: '209-555-5555',
-      website: 'https://www.chicken.coop/',
-      tags: ['meat', 'chickens', 'non-GMO', 'poultry', 'cooperative'],
-      mission: 'Coop with Coops',
-      description:
-        "Our chickens are grass-fed and produce the highest quality Grade AA eggs. Our quality of meat can't be beat!",
-      profilePicture: heihei,
-      instagram: 'https://instagram.com',
-      facebook: 'https://facebook.com',
-    },
-  ];
+  const [coops, setCoops] = React.useState([]); //all coops
+  //tracker for the starred coops, an array of starred coops
+  const [starredCoops, setStarredCoops] = React.useState([]);
+  const starredIds = null;
+  const [coopShown, setCoopShown] = React.useState([]);
+
+  async function fetchData() {
+    const res = await axios.get('/api/coops');
+    setCoops(res.data);
+    setCoopShown(res.data[0]);
+
+    //get the toggle star info
+    const starred = await axios.get('/api/getStarred/1')
+    //set the query data as the starred coops
+    let starredIds = starred.data.map((e) => {
+      return e.starred_coop_id
+    })
+    setStarredCoops(starredIds);
+  }
+
+  React.useEffect(() => {
+    fetchData();
+  }, []);
+
+  if (!coops) {
+    return <div>Loading...</div>;
+  }
+
+  
+  function toggleStar(starredId, starrerId) {
+    if (starredCoops.includes(starredId)) {
+      //if the coop is already starred
+      //remove the coop from the list of starred
+      console.log("HERE")
+      const index = starredCoops.indexOf(starredId);
+      if (index > -1) {
+        starredCoops.splice(index, 1);
+      }
+      //remove the row from the database
+      axios.delete('/api/delete', {
+        data: {
+          starred_id: starredId,
+          starrer_id: starrerId,
+        },
+      });
+    } else {
+      //if the coop isn't starred yet
+      //add the coop from the list of starred
+      starredCoops.push(starredId);
+      //make a post request
+      axios.post('/api/addStar', {
+        data: {
+          starred_id: starredId,
+          starrer_id: starrerId,
+        },
```

Comment:
again, if you're using a an additional data key, make sure you properly handle that in your router.post